src/Libraries/RevitNodesUI/Selection.cs

```diff
@@ -706,6 +707,16 @@ protected override IEnumerable<Element> ExtractSelectionResults(DividedSurface s
                 RevitElementSelectionHelper<DividedSurface>.GetFamilyInstancesFromDividedSurface(
                     selection);
         }
+
+        public override string ToString()
```

Comment:
This will not be transferred to the warning tooltip. It is to be displayed right in the node.